policy/README.md

```diff
@@ -180,6 +181,31 @@ The optional `end_date` field applies to all policies represented in the file.
 }
 ```
 
+[Top][toc]
+
+## Geography-Driven Events **[Beta feature](/general-information.md#beta-features):** *Yes (as of 1.1.0)*
```

Comment:
Since this info is relevant to both Agency and Provider, I think it should be moved to the general-information.md file and linked to from both APIs as needed.

Reply:
Hmm, I considered this but thought that it made sense to put it in Policy for now because Geography-Driven Events is exclusive to Policy. I was imagining it could move to General Information if/when Geography-Driven Events become usable in other ways (such as with non-Policy geographies and/or Metrics).

Reply:
I'm leaning General because that's where it's going to end up anyway.  IMHO.

Reply:
Happy to move it if that's the consensus here. If so, I think it would probably be good to add a bit of extra language making it clear that Geography-Driven Events are currently only issued-for and used-with Geometries that are tied to one or more active Policy rules.